blaze-core/src/main/scala/org/http4s/blaze/websocket/Http4sWSStage.scala

```diff
@@ -90,24 +81,24 @@ class Http4sWSStage(ws: ws4s.Websocket) extends TailStage[WebSocketFrame] {
         logger.trace(t)("WebSocket Exception")
         sendOutboundCommand(Command.Disconnect)
     }
-
-    ws.exchange.read.through(sink).run.runAsync(onFinish)
+    
+    (alive.discrete.map(!_)).wye(ws.exchange.read.to(snk))(wye.interrupt).run.runAsync(onFinish)
```

Comment:
Could we flip the boolean meaning from `alive` to `dead` or something so we don't need to do a `map` here? It seems wasteful to need to create all those objects just to do a negation.